src/components/Calculator.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable react/jsx-key */
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable-next-line react/jsx-key */
```

Comment:
👍 You did a great job by commenting your code in order to avoid pure-function linter errors!